public/css/login.css

```diff
@@ -1,3 +1,84 @@
-h1{
-  color: red;
+*,html{
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+}
+body{
+  background: #dddddd3b;
+}
+.left {
+  width: 50%;
+  float: right;
+  padding: 0;
+  margin: 0;
+  height: 100vh;
+  font-family: Arial, Helvetica, sans-serif;
+}
+
+.loginForm {
+  position: absolute;
+    width: 300px;
+    height: 450px;
+    top: 50%;
+    left: 50%;
+    transform: translate(50%,-50%);
+    box-sizing: border-box
+}
+.login {
+  width: 100px;
+  height: 100px;
+  margin: 0 auto;
+}
+.login i {
+  font-size: 100px;
+  color: #0F9CAF;
+}
+.form {
+  margin: 40px 0;
+}
+label {
+  margin: 0;
+  padding: 0;
+  font-weight: bold;
+  color: #333;
+}
+.form input {
+  width: 100%;
+  margin-top: 5px;
+  margin-bottom: 20px;
+  border: 1px solid #ddd;
+  outline: none;
+  height: 40px;
+  font-size: 16px;
+  padding-left: 7px;
+  border-radius: 4px
+}
+button {
+  width: 100%;
+  height: 40px;
+  cursor: pointer;
+  background: #0F9CAF;
+  border: none;
+  font-size: 18px;
+  color: white;
+  margin: 15px 0;
+  border-radius: 4px
+}
+button:hover {
+  background: rgb(3, 131, 148);
+}
+p {
+  color: #333;
+}
+
+a {
+  color: #0F9CAF;
+  text-decoration: none;
+  font-size: 16px;
+  font-weight: bold;
+}
+a:hover {
+  color: rgb(3, 131, 148);
+  transition: .2s;
```

Comment:
@AhmedAl-Almi It's best if your `transition` styles always specified which attribute you want the browser to transition (so here, it would be `transition: color .2s`). 
This will have better performance (discussed [here](https://stackoverflow.com/questions/8947441/css3-transitions-is-transition-all-slower-than-transition-x)) because the browser isn't trying to transition things that the user can't see.
That will also prevent you from seeing things animate that you didn't expect to animate (if the user changes the browser zoom or the page layout changes and this element changes in size, you probably don't want to see that change animate)

Reply:
This is a good idea, I like it.